examples/time_frequency/plot_time_frequency_sensors.py

```diff
@@ -56,11 +56,12 @@
 fig, axis = plt.subplots(1, 2, figsize=(7, 4))
 power.plot_topomap(ch_type='grad', tmin=0.5, tmax=1.5, fmin=8, fmax=12,
                    baseline=(-0.5, 0), mode='logratio', axes=axis[0],
-                   title='Alpha', vmax=0.45)
+                   title='Alpha', vmax=0.45, show=False)
 power.plot_topomap(ch_type='grad', tmin=0.5, tmax=1.5, fmin=13, fmax=25,
                    baseline=(-0.5, 0), mode='logratio', axes=axis[1],
-                   title='Beta', vmax=0.45)
+                   title='Beta', vmax=0.45, show=False)
 mne.viz.tight_layout()
+plt.show()
```

Comment:
The viz function is called twice, so we need to add `show=False` to have both plots in the same figure. 

If one wants to fix the layout, `show()` can only be called later.